src/model/CoilCoolingDXSingleSpeed.cpp

```diff
@@ -897,18 +923,21 @@ namespace model {
     setEnergyInputRatioFunctionOfFlowFractionCurve(energyInputRatioFofFlow);
     setPartLoadFractionCorrelationCurve(partLoadFraction);
     setEvaporativeCondenserEffectiveness(0.0);
-    setEvaporativeCondenserAirFlowRate(boost::none);
-    setEvaporativeCondenserPumpRatedPowerConsumption(boost::none);
+    autosizeEvaporativeCondenserAirFlowRate();                //autosize
+    autosizeEvaporativeCondenserPumpRatedPowerConsumption();  //autosize
     setCrankcaseHeaterCapacity(0.0);
     setMaximumOutdoorDryBulbTemperatureForCrankcaseHeaterOperation(0.0);
     setMinimumOutdoorDryBulbTemperatureforCompressorOperation(-25.0);  // Per E+ IDD default
-    //setSupplyWaterStorageTankName("");
-    //setCondensateCollectionWaterStorageTankName("");
+    setNominalTimeForCondensateRemovalToBegin(0.0);
+    setRatioOfInitialMoistureEvaporationRateAndSteadyStateLatentCapacity(0.0);
+    setMaximumCyclingRate(0.0);
+    setLatentCapacityTimeConstant(0.0);
+    setCondenserType("AirCooled");
+    setEvaporativeCondenserEffectiveness(0.9);
```

Comment:
You're changing a default of 0.0 to a default fo 0.9 here...  Yet you're not doing the same in CoilCoolingDXTwoSpeed.

Reply:
This is also called twice, oone with 0.0 above (historical) and one with 0.9 now

Reply:
Changing is a default means the regression tests may need to be adjusted to hardsize that field to the historical value or they'll show a deviation...